runtime/lua/vim/health.lua

```diff
@@ -341,6 +345,22 @@ M._complete = function()
   return rv
 end
 
+local function build_summary()
+  local s = ''
+  local errors = report_output['error'] or 0
+  local warns = report_output['warn'] or 0
+  local oks = report_output['ok'] or 0
+
+  -- s = s .. (errors > 0 and ('%d E'):format(errors) or '')
+  -- s = s .. (warns > 0 and (' %d W'):format(warns) or '')
+  -- s = s .. (oks > 0 and (' %d O'):format(oks) or '')
+  s = s .. (errors > 0 and ('%d ❌'):format(errors) or '')
+  s = s .. (warns > 0 and (' %d ⚠️'):format(warns) or '')
+  s = s .. (oks > 0 and (' %d ✅'):format(oks) or '')
```

Comment:
This can also be done implicit (if it's `nil` that means there are no messages of that type). Thus `or 0` and `> 0` can be removed. What is preferred?

Reply:
I'm fine with implicit.